cms/static/cms/js/main.js

```diff
@@ -25,23 +25,24 @@
                 dataType: 'json'
             });
             $(document).ajaxError(function(event, jqXHR, ajaxSettings) {
-                var message, msg;
+                var msg, contentType,
+                    message = gettext('This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.');  // eslint-disable-line max-len
                 if (ajaxSettings.notifyOnError === false) {
                     return;
                 }
-                if (jqXHR.responseText) {
-                    try {
-                        message = JSON.parse(jqXHR.responseText).error;
-                    } catch (error) {
-                        message = str.truncate(jqXHR.responseText, 300);
-                    }
-                } else {
-                    message = gettext('This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.');  // eslint-disable-line max-len
+                contentType = jqXHR.getResponseHeader('content-type');
+                if (contentType && contentType.indexOf('json') > -1 && jqXHR.responseText) {
+                    message = JSON.parse(jqXHR.responseText).error;
                 }
                 msg = new NotificationView.Error({
                     'title': gettext("Studio's having trouble saving your work"),
                     'message': message
                 });
+                Logger.log('edx.studio.ajax_error', {
```

Comment:
I think the error is probably already logged on the server, so it may be enough to just use ``console.log`` here. Using ``Logger.log`` means that the event gets reported to Segment, and then ends up in our analytics pipeline. I can't see us wanting to do analytics on this particular event.